docs/tempo/website/setup/tanka.md

```diff
@@ -0,0 +1,367 @@
+        requests:
+          storage: 50Gi
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: minio
+    spec:
+      selector:
+        matchLabels:
+          app: minio
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            # Label is used as selector in the service.
+            app: minio
+        spec:
+          # Refer to the PVC created earlier
+          volumes:
+            - name: storage
+              persistentVolumeClaim:
+                # Name of the PVC created earlier
+                claimName: minio-pv-claim
+          initContainers:
+            - name: create-buckets
+              image: busybox:1.28
+              command:
+                - "sh"
+                - "-c"
+                - "mkdir -p /storage/tempo-data"
+              volumeMounts:
+                - name: storage # must match the volume name, above
+                  mountPath: "/storage"
+          containers:
+            - name: minio
+              # Pulls the default Minio image from Docker Hub
+              image: minio/minio:latest
+              args:
+                - server
+                - /storage
+                - --console-address
+                - ":9001"
+              env:
+                # Minio access key and secret key
+                - name: MINIO_ACCESS_KEY
+                  value: "minio"
+                - name: MINIO_SECRET_KEY
+                  value: "minio123"
+              ports:
+                - containerPort: 9000
+                - containerPort: 9001
+              volumeMounts:
+                - name: storage # must match the volume name, above
+                  mountPath: "/storage"
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: minio
+    spec:
+      type: ClusterIP
+      ports:
+        - port: 9000
+          targetPort: 9000
+          protocol: TCP
+          name: api
+        - port: 9001
+          targetPort: 9001
+          protocol: TCP
+          name: console
+      selector:
+        app: minio
+    ```
+
+  1. Run the following command to apply the minio.yaml file:
+
+        ```bash
+        kubectl apply --namespace tempo -f minio.yaml
+        ```
+
+  1. To check that MinIO is correctly configured, sign in to MinIO and verify that a bucket has been created. Without these buckets, no data will be stored.
+     1.  Port-forward MinIO to port 9001:
+
+         ```bash
+          kubectl port-forward --namespace tempo service/minio 9001:9001
+          ```
+     1. Navigate to the MinIO admin bash using your browser: `https://localhost:9001`. The sign-in credentials are username `minio` and password `minio123`.
+     1. Verify that the Buckets page lists `tempo-data`.
+
+1. Configure the Tempo cluster using the MinIO object storage by updating the contents of the `environments/tempo/main.jsonnet` file by running the following command:
+
+   ```bash
```

Comment:
General philosophy question, is it better to have these large `cat EOF` blocks, or perhaps tell users to create a file with the following contents?  I ask because I think `jsonnet` is a supported code fence, so looking at the block would have some highlighting which might be nicer to look at.  Though perhaps the `jsonnet` fence is only supported on github?

Reply:
I'll try changing it to jsonnet and we'll see what happens when I build it locally.